aerleon/lib/openconfig.py

```diff
@@ -148,6 +163,17 @@ def ConvertToDict(
             protos = ['none']
 
         ace_dict = copy.deepcopy(term_dict)
+
+        # options
+        if self.term.option:
+            for opt in [str(x) for x in self.term.option]:
```

Comment:
Could this be simplified to `if self.term.protocol != ['tcp'] and 'tcp-established' in self.term.option `?
This should short circuit in the cases where the first option is false which is quick and with my other comment I don't believe options are not strings.

Reply:
I initially copied the code from the Juniper generator, the idea was that right now we only have 1 option, but the code needs to be ready to support more options. That said I don't think itterating over all the "options" are useful, but checking if a given option is in the array is enough (and cleaner). Let me know what you think of this new version.